### PR TITLE
test(app): add FileCard tests and update error copy

### PR DESCRIPTION
**Pull request by ncdiehl11**
closes [AUTH-517](https://opentrons.atlassian.net/browse/AUTH-517)

# Overview

add FileCard tests and update error copy

# Changelog

- add test file for new `FileCard` component
- update copy for incorrect file type error message on slideout

# Risk assessment

low

[AUTH-517]: https://opentrons.atlassian.net/browse/AUTH-517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/organisms/ChooseRobotSlideout/FileCard.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ export function FileCard(props: FileCardProps): JSX.Element {
         </StyledText>
         <Flex alignItems={ALIGN_CENTER}>
           <Btn
+            data-testid="x_btn"
```
Comment (koji) [inline app/src/organisms/ChooseRobotSlideout/FileCard.tsx]: I think we avoid using btn as much as possible.
```suggestion
            data-testid="close_button"
```

File: app/src/organisms/ChooseRobotSlideout/FileCard.tsx
Comment (koji) [inline app/src/organisms/ChooseRobotSlideout/FileCard.tsx:1]: @ncdiehl11 
Could you update the error message for non-csv file?
https://www.figma.com/design/MlZjOO8aZXsw3HfTBoyw9U/Release%3A-CSV?node-id=4-2967&t=CZik17e5cdohrL04-0#826144154

**Review by koji — APPROVED** [review]
test looks good to me!
thank you for adding the test.

File: app/src/organisms/ChooseRobotSlideout/FileCard.tsx
Comment (ncdiehl11) [reply app/src/organisms/ChooseRobotSlideout/FileCard.tsx:1]: oops, that change wasn't committed somehow. just pushed another commit

**Pull request merged**